R/merge_sce_list.R

```diff
@@ -256,11 +256,11 @@ merge_sce_list <- function(
       assay(merged_sce, name) <- as(assay(merged_sce, name), "CsparseMatrix")
     }
 
-    # now for the alt exps
+    # now for the alt exps, but leave them dense
     for(altexp_name in names(altexp_attributes)){
       alt_merged_sce <- altExp(merged_sce, altexp_name)
       for(name in assayNames(alt_merged_sce)){
-        assay(alt_merged_sce, name) <- as(assay(alt_merged_sce, name), "CsparseMatrix")
+        assay(alt_merged_sce, name) <- as(assay(alt_merged_sce, name), "Matrix")
```

Comment:
Should we be concerned about the amount of memory this might take up in a merged object if we leave them dense?

Reply:
I just saw your latest commit so ignore this comment.